2. Services/2.3 Orchestrations/2.3 Orchestrations.md

```diff
@@ -720,23 +720,24 @@ public partial class StudentOrchestrationService
         }
         catch (StudentValidationException studentValidationException)
         {
-            throw CreateAndLogDependencyValidationException(studentValidationException);
+            throw await CreateAndLogDependencyValidationExceptionAsync(studentValidationException);
         }
         catch (StudentDependencyValidationException studentDependencyValidationException)
         {
-            throw CreateAndLogDependencyValidationException(studentDependencyValidationException);
+            throw await CreateAndLogDependencyValidationExceptionAsync(studentDependencyValidationException);
         }
         catch (LibraryCardValidationException libraryCardValidationException)
         {
-            throw CreateAndLogDependencyValidationException(libraryCardValidationException);
+            throw await CreateAndLogDependencyValidationExceptionAsync(libraryCardValidationException);
         }
         catch (LibraryCardDependencyValidationException libraryCardDependencyValidationException)
         {
-            throw CreateAndLogDependencyValidationException(libraryCardDependencyValidationException);
+            throw await CreateAndLogDependencyValidationExceptionAsync(libraryCardDependencyValidationException);
         }
     }
 
-    private StudentOrchestrationDependencyValidationException CreateAndLogDependencyValidationException(Xeption exception)
+    private async ValueTask<StudentOrchestrationDependencyValidationException>
```

Comment:
need the await on line 747:
`        await this.loggingBroker.LogErrorAsync(studentOrchestrationDependencyValidationException);`

Reply:
Fixed